usrsctplib/netinet/sctputil.c

```diff
@@ -1703,7 +1704,7 @@ sctp_timeout_handler(void *t)
 	}
 	type = tmr->type;
 	tmr->stopped_from = 0xa005;
-	SCTPDBG(SCTP_DEBUG_TIMER1, "Timer type %d goes off\n", type);
+	SCTPDBG(SCTP_DEBUG_TIMER1, "Timer %p of type %d goes off with callout is %p\n", tmr, type, &(tmr->timer));
```

Comment:
Note for reviewers: During development of deadlock fix I've made several stupid mistakes, so having more detailed logs related to timer proven to be helpful to find and fix those mistakes - here the relationship between `struct sctp_timer *` and `struct sctp_callout *` is explicitly logged.